framework/materials/multi_group_xs/multi_group_xs.h

```diff
@@ -169,6 +176,24 @@ class MultiGroupXS : public MaterialProperty
   void ComputeDiffusionParameters();
 
   void TransposeTransferAndProduction();
+
+  // Check vector for all non-negative values
```

Comment:
If these are supposed to be for doxygen, then it needs to be `///`

Reply:
I hadn't intended that, but it's not a bad idea. Fixed.